conversation/ui
# Delete the Autocomplete component from the design system

**Why?**

The `Autocomplete` component was never used.  It's clear that it fell into oblivion, as its `downshift` dependency was still fixed on version `3.2.7`, released three years ago.

Let's delete this component, simplifying the design system and allowing apps to have non-conflicting `downshift` versions.

**Manual testing steps**

- Checkout this branch
- Install modules (e.g.: `make node_modules`)
- Build storybook (e.g.: `make storybook`)

When you open storybook's server, you'll see that the `Autocomplete` component is gone.
